guvnor-services/guvnor-services-backend/src/main/java/org/guvnor/common/services/backend/metadata/attribute/GeneratedFileAttributes.java

```diff
@@ -0,0 +1,11 @@
+package org.guvnor.common.services.backend.metadata.attribute;
+
+import org.uberfire.java.nio.file.attribute.BasicFileAttributes;
+
+/**
+ * Represents attributes of a file which has been generated.
+ */
+public interface GeneratedFileAttributes extends BasicFileAttributes {
```

Comment:
I wonder whether we should extend ```BasicFileAttributes``` as (as far as I can tell) the "basic" view is still used in addition to the "generated" view - and hence you're duplicating many attributes. Is there not a _higher_ super interface in the hierarchy you can extend that provides the "View" definition without duplicating attributes?